### PR TITLE
feat(webhook): plugin webhook's detail and list api can get api key info now

### PR DESCRIPTION
**Pull request by d4x1**
### ⚠️ Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing Documentation](https://devlake.apache.org/community/).
- [x] I have added relevant tests.
- [x] I have added relevant documentation.
- [x] I will add labels to the PR, such as `pr-type/bug-fix`, `pr-type/feature-development`, etc.

### Summary
What does this PR do?
1. add ApiKey field for plugin webhook's list and detail APIs, in order to make it possible to regenerabe webhook's api key with the api `HTTP PUT/api-keys/:apiKeyId/`.
2. Simplify the response struct to make code less.
3. add fields `postIssuesEndpoint` `closeIssuesEndpoint` `postPipelineTaskEndpoint` `postPipelineDeployTaskEndpoint` `closePipelineEndpoint`  to help to simplify the front end codes.

### Does this close any open issues?
Closes xx

### Screenshots
Include any relevant screenshots here.
1.
image
image

3. 
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: backend/plugins/webhook/api/connection.go [hunk 1]
```diff
@@ -150,9 +153,13 @@ func ListConnections(input *plugin.ApiResourceInput) (*plugin.ApiResourceOutput,
 	if err != nil {
 		return nil, err
 	}
-	responseList := []WebhookConnectionResponse{}
+	var responseList []*WebhookConnectionResponse
 	for _, connection := range connections {
-		responseList = append(responseList, *formatConnection(&connection))
+		webhookConnectionResponse, err := formatConnection(&connection, true)
```
Comment (klesh) [inline backend/plugins/webhook/api/connection.go:158]: I think we should delete the `api key hash` from the response to reduce the attach surface.

Comment (d4x1) [reply backend/plugins/webhook/api/connection.go:158]: Got it.

Comment (d4x1) [reply backend/plugins/webhook/api/connection.go:158]: Updated. Please review again. @klesh

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**